zk/conn.go

```diff
@@ -143,6 +150,9 @@ func ConnectWithDialer(servers []string, sessionTimeout time.Duration, dialer Di
 	if dialer == nil {
 		dialer = net.DialTimeout
 	}
+
+	newLogger := log.New(os.Stderr, "", log.LstdFlags)
```

Comment:
I think it would be best to use the default logger so that if someone has already used `log.SetOutput` then it'll work as expected (as before).

```
type defaultLogger struct{}

func (defaultLogger) Printf(format string, v ...interface{}) {
	log.Printf(format, v...)
}
```

```
newLogger := defaultLogger{}
```